Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -10,13 +10,13 @@ final class LoginViewController: UIViewController {
     @IBOutlet private(set) weak var passwordTextField: UITextField!
     @IBOutlet private(set) weak var forgotPasswordButton: UIButton!
     @IBOutlet private(set) weak var signInButton: UIButton!
-    @IBOutlet private(set) weak var signInWithGoogleButton: UIButton!
     @IBOutlet private(set) weak var signInWithFacebookButton: UIButton!
     @IBOutlet private(set) weak var signUpButton: UIButton!
 
     // MARK: Properties
 
     private let viewModel: LoginViewModelType
+    private var authService: Authentication!
```

Comment:
- Should be added to the view model. 
- This is a bad practice... Suggested 
```suggestion
    private lazy var authService: Authentication = GooglSignInAuthenticatorAdapter()
```